### DISCUSSION TITLE
Potential abuse of vercel.app domains by scammers

### DISCUSSION THREAD

**Discussion by silasabbott** [vercel/community#3576] [Help] [answered]
I've seen these fake posts on Facebook for a while, but just recently noticed they are linking to a bunch of different free Vercel domains. Just wanted you guys to be aware, not sure how you handle this internally

![image](https://github.com/vercel/community/assets/8752644/8eef0d0a-be96-435c-a10f-3e190cafab67)

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. If you haven't already, please also email [abuse@vercel.com](mailto:abuse@vercel.com) where the request will be investigated further. When doing this, please provide as much relevant information as possible 🙏

**Reply by st333v** [reply]
in my experience, vercel do not do anything when spam using vercel.app domains is reported. This problem is huge and has been going on for many months now, my facebook Page gets tons of comment spam of a gross image pretending to be a video link, and a link to a random vercel.app domain, i have put vercel.app into my Page's moderation tool so it auto hides any comment mentioning vercel.app but the comments still screw with comment moderation even though they are hidden, as it screws up the view comments by most recent option, as the comments with spammy vercel.app replies go to the top of the most recent comments list, thus making it hard to find the real most recent comments. 

Heres a screenshot example of the crap they are spamming:
[removed by @Pieparker – please see our [Code of Conduct](https://github.com/vercel/community/blob/main/.github/CODE_OF_CONDUCT.md)]


when i reported this issue to Vercel back in march heres the reply i received from them, (they seem to want a police report or something similar before they will take action lol):

-------------
Hi there,

We have investigated this matter and cannot confirm that the site(s) violate our Terms of Service.

Online abuse takes many forms, and we may not be familiar with the reported abuse vector. Therefore, more evidence is required if you want us to investigate this further.

A related government notice, legal notice, or police report naming the site(s) as fraudulent may be necessary.

Please let us know if you have any further questions or issues!

Kind regards,
Rob

**Reply by Pieparker** [reply]
Hey @st333v, we review every abuse report that we receive. We are currently working through a sizable backlog. If you are unhappy with the outcome of a review, I would advise you to engage with our abuse team by replying to them in your report.

[N.B. Posts within the Vercel Community Discussions must adhere to our [Code of Conduct](https://github.com/vercel/community/blob/main/.github/CODE_OF_CONDUCT.md)].

**Comment by grblman** [comment]
<img width="598" alt="Capture d’écran 2023-11-19 à 11 19 27" src="https://github.com/vercel/community/assets/55132830/6f43da53-f979-4624-870b-9d40ff457233">

**Reply by grblman** [reply]
this link it is ok?

**Reply by amyegan** [reply]
If you believe that a Vercel deployment contains malicious or illegal content, please report it to our abuse team by sending a message to [abuse@vercel.com](mailto:abuse@vercel.com). Be sure to provide as much relevant information as possible in your message 🙏
